config/checker/com.intellij.astub

```diff
@@ -210,6 +210,13 @@ package com.intellij.openapi.project;
 @UIPackage package com.intellij.openapi.roots.ui.configuration.libraryEditor;
 @UIPackage package com.intellij.openapi.roots.ui.util;
 @UIPackage package com.intellij.openapi.ui;
+
+class DialogWrapper {
+  // `this` in the below method is @UnknownInitialization since this methods is called from DialogWrapper's c'tor
+  @Nullable javax.swing.JRootPane getRootPane(@UnknownInitialization DialogWrapper this);
```

Comment:
Sorry @mkondratek if/that I suggested to remove that before - as a rule of the thumb, any method in any class that's invoked from any of this class's c'tors should be marked as `@UnknownInitialization ... this`